src/components/search/data/hooks.js

```diff
@@ -106,3 +106,20 @@ export const useNbHitsFromSearchResults = (searchResults) => {
 
   return nbHits;
 };
+
+export const useDefaultSearchFilters = ({ enterpriseConfig, subscriptionPlan }) => {
+  const filters = useMemo(
```

Comment:
Can you talk a bit about why you used `useMemo` here? I'm fairly new to hooks and I want to get a better idea of when to use this one and why.

Reply:
@raq929 The way I think about when to use `useMemo` is by asking: "Do I need to compute a value that only needs to change when dependencies of that value change?"

If the answer is "yes", I'll use the `useMemo` hook.

In this case, I want to essentially re-compute `filters` anytime `subscriptionPlan` or `enterpriseConfig` change. But if those dependencies don't change, I don't want to have to run through the logic on every component re-render; rather, I only want to compute `filters` when needed.